Example/PrebidDemo/PrebidDemoSwift/AppDelegate.swift

```diff
@@ -37,6 +37,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         Prebid.shared.prebidServerHost = PrebidHost.Appnexus
         Prebid.shared.prebidServerAccountId = "bfa84af2-bd16-4d35-96ad-31c6bb888df0"
+        
+        // User Id from External Third Party Sources
+        var externalUserIdArray = [ExternalUserId]()
+        externalUserIdArray.append(ExternalUserId(source: "adserver.org", userIdArray: [["id" : "111111111111", "ext" : ["rtiPartner" : "TDID"]]]))
```

Comment:
I would create a UserID class with type, id & ext as it objects & get the information from the publisher then add it to the external userID array. This resolves ambiguity of what needs to be added...

Reply:
Okay, I thought that publisher can add any JSON type not just only these three types "atype", "id", "ext".  
So please let me know if these three types are fixed then we can move with UserID class implementation as well else the current implementation seems more flexible.

Reply:
ext is the only field then user can add any json. all others are pretty standard. Also in future if any other params gets added we can add the same to the class

Reply:
done